src/nodes/Table/TableCell.js

```diff
@@ -30,4 +32,32 @@ export default TableCell.extend({
 			},
 		}
 	},
+
+	addProseMirrorPlugins() {
+		return [
+			new Plugin({
+				props: {
+					handlePaste: (view, event, slice) => {
+						if (slice.content.childCount > 1) {
+							const state = view.state
+							const childNodes = []
+							for (let i = 0; i < slice.content.childCount; i++) {
+								if (i) {
+									childNodes.push(state.schema.text('\n'))
+								}
+
+								if (!slice.content.child(i).firstChild) {
+									continue
```

Comment:
```suggestion
									// Ignore empty children (i.e. empty lines)
									continue
```

How about adding a comment here to document what happens here?